src/Foreign/NullOrUndefined.js

```diff
@@ -1,3 +1,3 @@
-exports['null'] = null;
+export const nullImpl = null
```

Comment:
Why not this?
``
export { null as null };
export { undefined as undefined };
``

Reply:
I didn't know this works

Reply:
And it doesn't. This causes problems when bundling with swc-bundler at least.

Reply:
That's a shame.